backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/UserController.kt

```diff
@@ -45,10 +42,16 @@ class UserController(val userRepository: UserRepository) {
       else -> Bioklubbnummer(newDetails.bioklubbnummer!!)
     }
 
+    val newForetagsbiljetter = when {
+      newDetails.foretagsbiljetter == null -> emptyList()
```

Comment:
not needed if type not nullable. Also, I think the `when` could be replaced with an elvis operator otherwise.